README.md

```diff
@@ -1,2 +1,76 @@
-# manifest
-The feeless order book exchange that supercharges your on-chain trading abilities
+# Manifest
+
+The feeless order book exchange that supercharges your on-chain trading
+abilities Manifest is the next generation of orderbooks on Solana. It is
+optimized for freedom. No more permissioned markets. No more trading fees. No
+more large 2 SOL rent to start a market. Capital efficiency built in.
```

Comment:
No more expensive rent or gas costs to start a market

Reply:
gas is gonna be higher for manifest to reserve capacity early on